### DISCUSSION TITLE
Defining branch and bound strategies

### DISCUSSION THREAD

**Discussion by LuxGiammi** [coin-or/python-mip#206] [New features]
Hi everybody,

I was wondering if python-mip can handle the definition of branch and bound strategies (like CyLP: *For example, you may define cut generators, branch-and-bound strategies, and primal/dual Simplex pivot rules completely in Python*). I know that it's possible to do such a thing in CBC (https://www.coin-or.org/Cbc/cbcuserguide.html examples 3.1, 3.2 and 3.3) and apparently CyLP allows to do the same with the method `setNodeCompare`.

I was looking for the same feature in python-mip, but I couldn't find anything similar in the documentation.

Does python-mip support this feature (or it's planning to)?

Thanks a lot

**Comment by tuliotoffolo** [comment]
Hi @LuxGiammi,

Python-MIP currently does not support this feature, but we plan to do so in the near future.  :)
